website/content/docs/concepts/lease-count-quota-exceeded.mdx

```diff
@@ -0,0 +1,42 @@
+---
+layout: docs
+page_title: Lease count quota exceeded
+description: |-
+  Vault Enterprise error when quota limit is reached.
+---
+
+# Lease count quota exceeded
+
+Vault returns a `429 - Too Many Requests` response if a new lease request
+violates the quota limit, to guard against [lease
+explosions](/vault/docs/concepts/lease-explosions).
+
+The error will appear as follows:
```

Comment:
```suggestion
```

Reply:
We don't really need a separate line to introduce the example, you can just put a colon at the end of the previous sentence and use that as the introduction.